src/TimeColumn.jsx

```diff
@@ -63,9 +65,18 @@ export default class TimeColumn extends Component {
       date = next
     }
 
+    const classes = cn(
+      this.props.className,
+      'rbc-time-column',
+      {
+        'rbc-today': dates.isToday(this.props.max),
```

Comment:
I think it might be enough to add the 'rbc-today' class, then folks can style that how they want without needing to add an additional prop. Does that still work for you?